cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+---
+CPS: ?
```

Comment:
```suggestion
CPS: 10
```
... all ready to rename the containing directory & update the "rendered" link to match. 🎉

Reply:
Yes!
All my CIPs and CPSs are multiples of 5 -- I have kept the pattern going